cabal.config

```diff
@@ -1,43 +1,37 @@
-constraints: array ==0.5.1.0,
-             base ==4.8.2.0,
+constraints: array ==0.5.2.0,
```

Comment:
If we're switching `kioku` development to `stack`, I would expect this file to go away. It's only used for local development using cabal.